releases/dogwood/3/fun/Dockerfile

```
@@ -215,12 +215,8 @@ RUN apt-get update && \
 
 RUN groupadd --gid ${DOCKER_GID} edx || \
       echo "Group with ID ${DOCKER_GID} already exists." && \
-    useradd \
-      --create-home \
-      --home-dir /home/edx \
-      --uid ${DOCKER_UID} \
-      --gid ${DOCKER_GID} \
-      edx && \
+    useradd --create-home --home-dir /home/edx --uid ${DOCKER_UID} --gid ${DOCKER_GID} edx || \
+      echo "User with ID ${DOCKER_UID} already exists." && \
```

We are not sure of the reason why the useradd command fails:

```suggestion
      echo "Skip user creation (user with ID ${DOCKER_UID} already exists?)" && \
```

---

Good point